console/st/console-st.sh

```diff
@@ -45,6 +44,14 @@ CONF_PROFILE='<0x7000000000000001:0>'
 NODE_UUID=02e94b88-19ab-4166-b26b-91b51f22ad91  # required by `common.sh'
 . $M0_SRC_DIR/m0t1fs/linux_kernel/st/common.sh  # modload_galois
 
+cmd=$(whereis fi_info | cut -d ':' -f2)
```

Comment:
Move this logic to m0t1fs/linux_kernel/st/common.sh

Also, add an option to run-st to specify the transport type, which will help in testing both the transports at the same time while in jenkins or running ST's in VM.